project/pimcore.py

```diff
@@ -0,0 +1,14 @@
+from .remote import RemoteProject
+
+
+class Pimcore5(RemoteProject):
+    major_version = 'v5.8.2'
```

Comment:
This shouldn't pin to an exact version.  Then it never auto-updates.  It should pin to `v5` if we want it to automatically pull in the latest 5.x version every time we run update, or to `v5.8` if we want it to only auto-update with the 5.8 series.

I don't know which is appropriate for Pimcore, but I'd assume `v5` is what we want?